src/main/scala/org/ergoplatform/local/NipopowVerifier.scala

```diff
@@ -8,20 +8,52 @@ import scorex.util.ModifierId
   * A verifier for PoPoW proofs. During its lifetime, it processes many proofs with the aim of deducing at any given
   * point what is the best (sub)chain rooted at the specified genesis.
   *
-  * @param genesisId    - the block id of the genesis block
+  * @param genesisIdOpt    - the block id of the genesis block as hard-coded in config. If not available, proof for
+  *                        any chain could be accepted! This is not desirable likely, thus better to set genesis block
+  *                        id in settings!
```

Comment:
This would be not convenient for running (local) testnets and other short-living networks - there is no genesis ID when then network is just generated, so what to do in this case?